specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2023-04-01/registries.json

```diff
@@ -0,0 +1,891 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Machine Learning Services",
```

Comment:
I see that registries already has an older version here : 
[https://github.com/komo23x/azure-rest-api-specs/tree/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview](https://github.com/komo23x/azure-rest-api-specs/tree/main/specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-12-01-preview)

please add the previous version as the first commit before creating a PR as explained in step 6 here : [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version)

Reply:
will do. should I abandon this PR and start a fresh one? or can I just add on a commit with the old registries version to this PR

Reply:
For now you can just add a commit with the old registries and add a comment with the id so that reviewers are aware which version to compare to